src/pkg/inspectors/workloadscanner/pkg/controller.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+package pkg
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/vmware-tanzu/cloud-native-security-inspector/src/api/v1alpha1"
+	wl "github.com/vmware-tanzu/cloud-native-security-inspector/src/lib/assets/workload"
+	"github.com/vmware-tanzu/cloud-native-security-inspector/src/lib/log"
+	exporter_inputs "github.com/vmware-tanzu/cloud-native-security-inspector/src/pkg/exporter/inputs"
+	itypes "github.com/vmware-tanzu/cloud-native-security-inspector/src/pkg/inspectors/imagescanner/types"
```

Comment:
You assessmentReport is from this pkg? Then why are your defining a new one in src/pkg/inspectors/workloadscanner/types/report.go？

Reply:
itypes.AssessmentReport on line 168